librz/bin/meson.build

```diff
@@ -141,7 +156,8 @@ rz_bin_sources = [
   'p/bin_xbe.c',
   'p/bin_xnu_kernelcache.c',
   'p/bin_xtr_fatmach0.c',
-  'p/bin_xtr_pemixed.c',
+  # this was never built.
+  # 'p/bin_xtr_pemixed.c',
```

Comment:
what do you mean that it was never built? if it was here it should have been built. I don't get it.

Reply:
the code was there, but it was never enabled as plugin. that is why when i added it back we had "test failing" but just because now was showing up.